src/org/opensolaris/opengrok/index/IndexDatabase.java

```diff
@@ -93,9 +96,17 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IndexDatabase.class);
 
+    /**
+     * Formerly, every delete issued an IndexWriter commit(). This is a first
+     * draft of a policy value used to flush -- not commit -- deleted Lucene
+     * items periodically.
+     */
+    private static final int MAX_BUFFERED_DELETE_TERMS = 200;
```

Comment:
would it make sense to expose this as tunable?

Reply:
I would say no. I do not imagine anyone would be able to definitively trace any performance issue to this setting — and compared to the baseline behavior of OpenGrok where it executes `commit()` on each deletion of document, any non-zero, but "reasonable" (i.e., not too far from zero) value here seems OK.